src/components/repo-detail.tsx

```diff
@@ -225,6 +227,26 @@ export function RepoDetail(props: RepoDetailProps) {
           </a>
         </div>
       )}
+
+      {Boolean(filename) && (
+        <div className="min-w-0 space-y-2">
+          <div className="mt-6 flex items-center bg-indigo-700 hover:bg-indigo-800 focus:bg-indigo-800 h-9 p-2 rounded text-white text-xs focus:outline-none focus:ring-2 focus:ring-indigo-400 w-full lg:max-w-md">
+            <label className="flex items-center space-x-2 w-full">
+              <input
+                checked={showSql}
+                onChange={(e) => {
+                  setShowSql(e.target.checked);
+                }}
+                className="text-indigo-700 hover:text-indigo-800 focus:text-indigo-800"
+                type="checkbox"
+              />
+              <span className="text-xs font-medium text-indigo-200 select-none">
+                Show SQL UI
```

Comment:
I like the idea of using a small widget as a toggle, like the checkbox in [c66c2a2](https://github.com/githubocto/flat-viewer/pull/39/commits/c66c2a2780f5d6c98dd51128c72029c362d9bbdc). With the right label it should feel almost intuitive to bring up, maybe..
```suggestion
                SQL Console
```